test/unit/foreman/renderer/snapshots/ProvisioningTemplate/iPXE/iPXE intermediate script.snap.txt

```diff
@@ -0,0 +1,76 @@
+isset ${net13/mac} || goto no_nic
+dhcp net13 || goto net14chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net13/mac} || goto net14:net14
+isset ${net14/mac} || goto no_nic
+dhcp net14 || goto net15chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net14/mac} || goto net15:net15
+isset ${net15/mac} || goto no_nic
+dhcp net15 || goto net16chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net15/mac} || goto net16:net16
+isset ${net16/mac} || goto no_nic
+dhcp net16 || goto net17chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net16/mac} || goto net17:net17
+isset ${net17/mac} || goto no_nic
+dhcp net17 || goto net18chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net17/mac} || goto net18:net18
+isset ${net18/mac} || goto no_nic
+dhcp net18 || goto net19chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net18/mac} || goto net19:net19
+isset ${net19/mac} || goto no_nic
+dhcp net19 || goto net20chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net19/mac} || goto net20:net20
+isset ${net20/mac} || goto no_nic
+dhcp net20 || goto net21chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net20/mac} || goto net21:net21
+isset ${net21/mac} || goto no_nic
+dhcp net21 || goto net22chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net21/mac} || goto net22:net22
+isset ${net22/mac} || goto no_nic
+dhcp net22 || goto net23chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net22/mac} || goto net23:net23
+isset ${net23/mac} || goto no_nic
+dhcp net23 || goto net24chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net23/mac} || goto net24:net24
+isset ${net24/mac} || goto no_nic
+dhcp net24 || goto net25chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net24/mac} || goto net25:net25
+isset ${net25/mac} || goto no_nic
+dhcp net25 || goto net26chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net25/mac} || goto net26:net26
+isset ${net26/mac} || goto no_nic
+dhcp net26 || goto net27chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net26/mac} || goto net27:net27
+isset ${net27/mac} || goto no_nic
+dhcp net27 || goto net28chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net27/mac} || goto net28:net28
+isset ${net28/mac} || goto no_nic
+dhcp net28 || goto net29chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net28/mac} || goto net29:net29
+isset ${net29/mac} || goto no_nic
+dhcp net29 || goto net30chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net29/mac} || goto net30:net30
+isset ${net30/mac} || goto no_nic
+dhcp net30 || goto net31chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net30/mac} || goto net31:net31
+isset ${net31/mac} || goto no_nic
+dhcp net31 || goto net32chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net31/mac} || goto net32:net32
+isset ${net32/mac} || goto no_nic
+dhcp net32 || goto net33chain http://foreman.some.host.fqdn/unattended/iPXE?mac=${net32/mac} || goto net33
```

Comment:
still a missing newline. `chain` needs to be a separate line.

Reply:
Ah, damnit. So I guess the rake take which generates this isn't working properly.

Reply:
Answering my own assumption: Nope, the template is incorrect, will adjust it